test/builder/pipelineRun_test.go

```diff
@@ -77,3 +79,151 @@ func Test_PipelineRunAbort(t *testing.T) {
 	assert.Equal(t, api.IntentAbort,
 		pipelineRun.Spec.Intent)
 }
+
+func Test_PipelineRunLoggingWithRunID(t *testing.T) {
+	buildID := uuid.New().String()
+	realmID := uuid.New().String()
+	pipelineRun := PipelineRun("prefix1", "namespace1",
+		PipelineRunSpec(
+			LoggingWithRunID(&api.CustomJSON{
+				map[string]string{
+					"jobId":   "1",
+					"buildId": buildID,
+					"realmId": realmID,
+				}}),
+		),
+	)
+	assert.DeepEqual(t, &api.Logging{
+		Elasticsearch: &api.Elasticsearch{
+			RunID: &api.CustomJSON{
+				map[string]string{
+					"jobId":   "1",
+					"buildId": buildID,
+					"realmId": realmID,
+				}},
+		},
+	}, pipelineRun.Spec.Logging)
+}
+
+func Test_PipelineRunLoggingIndexURL(t *testing.T) {
+	pipelineRun := PipelineRun("prefix1", "namespace1",
+		PipelineRunSpec(
+			LoggingWithIndexURL("testURL"),
+		),
+	)
+	assert.DeepEqual(t, &api.Logging{
+		Elasticsearch: &api.Elasticsearch{
+			IndexURL: "testURL",
+		},
+	}, pipelineRun.Spec.Logging)
+}
+
+func Test_PipelineRunLoggingWithRunIDAndWithIndexURL(t *testing.T) {
+	buildID := uuid.New().String()
+	realmID := uuid.New().String()
+	pipelineRun := PipelineRun("prefix1", "namespace1",
+		PipelineRunSpec(
+			LoggingWithIndexURL("testURL"),
+			LoggingWithRunID(&api.CustomJSON{
+				map[string]string{
+					"jobId":   "1",
+					"buildId": buildID,
+					"realmId": realmID,
+				}}),
+		),
+	)
+	assert.DeepEqual(t, &api.Logging{
+		Elasticsearch: &api.Elasticsearch{
+			IndexURL: "testURL",
+			RunID: &api.CustomJSON{
+				map[string]string{
+					"jobId":   "1",
+					"buildId": buildID,
+					"realmId": realmID,
+				}},
+		},
+	}, pipelineRun.Spec.Logging)
+}
+
+func Test_PipelineRunLoggingWithAllParams(t *testing.T) {
```

Comment:
Looks to me the same as Test_PipelineRunLoggingWithRunIDAndWithIndexURL
May be switch order of 
-LoggingWithIndexURL("testURL"),
- LoggingWithRunID(

to get a new test

Reply:
You are right. They are the same. I removed the second one.